content/blog/2020/2020-4-015-covid-19-part-3.md

```diff
@@ -0,0 +1,112 @@
++++
+author = "Andreja Kovačič"
+date = "2020-04-13"
+draft = false
+title = "Data Mining COVID-19 Epidemics: Part 3"
+type = "blog"
+thumbImage = "/blog_img/2020/2020-04-09-choropleth.png"
```

Comment:
This needs changing! It would be nice to have some line chart or something related to timeseries. Please note that the size has to be 300x200.